chapters/101.rst

```diff
@@ -22,9 +24,14 @@ Document
 	:width: 25%
 
 
-Documents are pieces of data in any dataset you want to search, and the input queries you use to find what you want. Basically, they are the input and output data for your search workflows.
+Documents are pieces of data in any dataset you want to search, and the input queries you use to find what you want. Basically, they are the input and output data for your search workflows, also used in Jina Flow.
+
+Document is a primitive data type in Jina. You can compare the document of Jina to `ndarray` of `numpy` or `tensor` of `tensorflow`. Document is also a powerful container for multimedia data. 
+A good example is the text we type into a search engine like Google. Documents can also be video clips, genetic sequences, MP3 files, scientific papers, funny GIFs, or many other things. You can hence store text, image, audio and the array inside the document. 
+
+Document can be recursive in two directions - a document has its nearest neighbors and neighbors can have their own nearest neighbors. This is considered as high order matches in Jina.
+In the other direction a document can be divided into sub-documents and the sub-document can be further segmented into finer granularities. This rich structure allows Jina to represent complex documents with hierarchies and multiple modalities.
```

Comment:
Can you ensure we use:
- **A** **D**ocument (use 'a' before "Document" and start the word with a capital D)